Cubical/Foundations/HLevels.agda

```diff
@@ -111,16 +114,92 @@ hLevelPi (suc (suc n)) h f g =
 isSetPi : ((x : A) → isSet (B x)) → isSet ((x : A) → B x)
 isSetPi Bset = hLevelPi 2 (λ a → Bset a)
 
+squeezeSq≡
```

Comment:
I think this should go into GroupoidLaws

Reply:
I can move it. I've been wondering though, GroupoidLaws often takes an extremely long time to type check for me. I actually don't know how long, because I've never waited for it to complete, just walked away and it was done an hour or two later. Does this not happen for other people?

It also only seems to happen when not loading the file directly, but as a dependency of another module.

Reply:
I think was related to some Agda bug. If you update your Agda installation this should go away, I've at least never experienced this for the last month or something.

Reply:
The whole library takes `	1m44.752s` to compile for me, on a Chromebook. Like mortberg says, reinstalling agda should fix this problem.

Reply:
eta equality for ```isEquiv``` were making it slow if you had the builtins for ```Glue``` loaded.

Reply:
Should `squeezeCu≡` (and `squeezeFace≡`) go in GroupoidLaws as well?

Reply:
I guess so. But I'm a bit confused why you need both isGroupoid1 and 2? Shouldn't it be sufficient to just have the most general one?

Reply:
I couldn't understand how to do it all in one step. I guess now that I figured it out I could collapse it into one definition.

Reply:
Sorry for the delay. I got rid of the middle `isGroupoid` type. Took me a while to figure out, since inlining seems to make it more difficult for Agda to infer all the metas (and it was difficult for me to infer them, as well :)).

I didn't move `squeezeCu≡` because its use of stuff from `Transport` causes a cycle. I could make it private if you'd prefer.